Documentation/MigrateToComposer/VersionControl.rst

```diff
@@ -82,10 +82,11 @@ While it can be tempting to just edit the :file:`composer.json` file manually,
 you should ideally use the proper :bash:`composer` commands to not introduce
 formatting errors or an invalid configuration.
 
-If you have regular maintenance automation (e.g. via `Dependabot <https://github.com/dependabot>`__),
-and your dependencies are locked to minor and patch-level releases, you could
-also use the :bash:`composer update` command, which would update all involved
-packages to their maximum allowed versions.
+You should avoid running :bash:`composer update` without specifying package names
+explicitly. You can use regular maintenance automation (e.g. via
```

Comment:
We avoid using "e.g." instead use "for example".

Reply:
Changed for this. There are many "e.g." in the other docs, let us address this in a different PR.